modules/ROOT/pages/apikit-tutorial.adoc

```diff
@@ -4,16 +4,100 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: apikit, raml, tutorial, api, rest
 
-In this procedure, you build and deploy an APIkit project that imports a RAML. You run the API in Anypoint Studio, use API Console to simulate calls to the API, prepare the project for deployment, deploy the project to Anypoint Platform, and apply a policy to the API.
+In this workflow, you build an APIkit project that imports a RAML. You run the API in Anypoint Studio and use API Console to simulate calls to the API.
 
 Studio 6.0 or later is recommended for this procedure. The API you create covers two main tasks:
 
 * Updating vending machine data: to update the central database on each sale
 * Tracking inventory: to find out if a machine is low on stock
 
-. xref:apikit-create.adoc[Create a new project].
-. xref:apikit-run-test.adoc[Run and test the project].
-. xref:apikit-add-logic.adoc[Add logic to the project].
-. xref:apikit-deploy.adoc[Deploy the project].
-. xref:apikit-apply-policy.adoc[Apply a policy].
-. xref:apikit-test-policy.adoc[Test a Policy].
\ No newline at end of file
+== Create a New APIkit Project
+
+This procedure launches the new project and generates a Main flow with an HTTP Listener, an APIkit Router, and references to flows to handle errors. The project support REST resources and methods. These flows, aptly called skeleton flows, are the bare minimum needed to support the RAML specification. The RAML file imported by this procedure resides in `src/main/api`.
+
+. Download the example file to your local drive:
++
+`+link:{attachmentsdir}/api.raml[api.raml]+`.
+. Under the File menu, select *New > Mule Project*.
+. In the New Mule Project wizard, enter a Name, myproject for example, Mule Runtime 3.8.0.
++
+. Check Add APIkit Components. In the RAML File field, browse to and select the downloaded `api.raml` file.
++
+image::apikit-tutorial-3e0e4.png[height=633,width=600]
++
+. Click *Finish*.
+
+== Run and Test the Project
+. Right-click the project name in the Project Explorer, then select *Run As > Mule Application*.
++
+The APIkit Console appears below the canvas, displaying the resource-action pairs exposed by the API.
++
+image::apikitconsole-tutorial.png[APIkitconsole-tutorial]
++
+. Click *Introduction* under Documentation to view the API description.
++
+The main Introduction title and sentence map to the contents of `documentation` in the root section of the RAML file.
++
+. Click `/machines` to expand the section. Expand `GET: /machines`, then click the tab labeled *Try It*.
```

Comment:
This step is done in the UI? Any time a user is clicking on something, use bold to indicate what they click on. So trade the tick marks for bold.
You can also make it shorter like:
Click **/machines** to expand the section, and expand **Get: /machines**, then click the **Try It** tab.